scripts/docgen/docgen.php

```diff
@@ -17,7 +17,7 @@
   |             Philip Olson <philip@php.net>                            |
   +----------------------------------------------------------------------+
  
-  $Id$
+  $Id: docgen.php 336015 2015-02-27 20:31:12Z aharvey $
```

Comment:
We still need this in the docs, right? :(

Reply:
I don't think that this is still needed (we need the IDs of the documentation XML files for the translations, though).

Reply:
Thanks, done.